src/pages/Detailed/mobile/MobileDetailedPage.js

```diff
@@ -112,7 +112,8 @@ const MobileDetailedPage = props => {
   ]
 
   if (extraMetric) {
-    const { alias } = Metrics[extraMetric.name]
+    const { alias } =
+      Metrics[extraMetric.name] || compatabilityMap[extraMetric.name]
```

Comment:
Duplicate of line 59
![image](https://user-images.githubusercontent.com/14061779/67072707-26cb0480-f18e-11e9-8c26-00f919e54571.png)